$MirOS: ports/print/cups/patches/patch-filter_imagetoraster_c,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- filter/imagetoraster.c.orig	Tue May  2 23:59:33 2006
+++ filter/imagetoraster.c	Tue May  9 17:39:40 2006
@@ -307,7 +307,7 @@ main(int  argc,				/* I - Number of comm
       close(mypipes[1]);
 
       execlp("pstoraster", argv[0], argv[1], argv[2], argv[3], argv[4], argv[5],
-             NULL);
+             (char *)NULL);
       perror("ERROR: Unable to exec pstoraster");
       return (errno);
     }
@@ -336,7 +336,7 @@ main(int  argc,				/* I - Number of comm
     */
 
     execlp("imagetops", argv[0], argv[1], argv[2], argv[3], argv[4], argv[5],
-           argv[6], NULL);
+           argv[6], (char *)NULL);
     perror("ERROR: Unable to exec imagetops");
     return (errno);
   }
